client/src/App.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { fireEvent, render, screen } from '@testing-library/react'
+import fakeKiosk from '../test/helpers/fakeKiosk'
+
+import App from './App'
+
+beforeEach(() => {
+  // we don't care about network errors logged to the console, they're crowding things
+  jest.spyOn(console, 'error').mockImplementation(() => {}) // eslint-disable-line @typescript-eslint/no-empty-function
```

Comment:
This is a bit of a code smell. Can we stop them from logging in the first place?

Reply:
we cannot stop it from logging the network error. Note that it doesn't cause the test to fail when the network error occurs, and because the network error *is* expected in this case (it's normal system operation), this feels like the best way to not make running tests look unnecessarily scary.